taskcluster/fenix_taskgraph/routes.py

```diff
@@ -12,6 +12,12 @@
 # In the future, notifying consumers may be easier (https://bugzilla.mozilla.org/show_bug.cgi?id=1548810), but
 # we need to remember to tell users for the time being
 SIGNING_ROUTE_TEMPLATES = [
+    "index.{trust-domain}.v2.{project}.{variant}.latest.{abi}",
```

Comment:
This matches what android-components is about to generate: https://github.com/mozilla-mobile/android-components/pull/6724/files#diff-6aa94c011deefaa63bc79f096403142fR12

And this is closer to https://searchfox.org/mozilla-central/rev/d2cec90777d573585f8477d5170892e5dcdfb0ab/taskcluster/taskgraph/transforms/task.py#248-253